Demos/Aspose.Email.Cloud.Live.Demos/Aspose.Email.Cloud.Live.Demos.UI/Services/AsposeEmailCloudApiService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Aspose.Email.Cloud.Sdk.Api;
+using Aspose.Email.Cloud.Sdk.Model;
+using System.IO;
+
+namespace Aspose.Email.Cloud.Live.Demos.UI.Services
+{
+    public class AsposeEmailCloudApiService
+    {
+        public Stream ConversionByEmailCloudApi(EmailCloud AsposeEmailCloudApi, Stream file, string fromFormat, string toFormat)
```

Comment:
The method name should be just `Convert`.

EmailCloud instance should be obtained in the constructor using dependency injection.